### ISSUE TITLE
TypeError: load() missing 1 required positional argument: 'Loader'

### ISSUE THREAD

**Issue by germainlefebvre4** [thedavidwhiteside/ansible-docgen#40]
The error `TypeError: load() missing 1 required positional argument: 'Loader'` comes up at running the docgen on ansible directory (playbook dir or role dir).

### Context

Dockerfile:
```Dockerfile
FROM python:3.9-slim
RUN pip install ansible-docgen
WORKDIR /app
```

Run docker image:
```bash
docker run -ti --rm -v $PWD:/app ansible-docgen:latest ansible-docgen -p ansible-playbooks/roles/myrole
```

Error stacktrace:
```
Traceback (most recent call last):
  File "/usr/local/bin/ansible-docgen", line 7, in <module>
    cli.run()
  File "/usr/local/lib/python3.9/site-packages/ansibledocgen/cli.py", line 37, in run
    self.dirparser = DirParser(project)
  File "/usr/local/lib/python3.9/site-packages/ansibledocgen/parser/dir.py", line 20, in __init__
    self.playbookparser.parse_playbooks()
  File "/usr/local/lib/python3.9/site-packages/ansibledocgen/parser/playbook.py", line 20, in parse_playbooks
    self.parse_playbook(playbook)
  File "/usr/local/lib/python3.9/site-packages/ansibledocgen/parser/playbook.py", line 52, in parse_playbook
    yamldata = yaml.load(data)
TypeError: load() missing 1 required positional argument: 'Loader'
```

### How to reproduce?
```bash
docker build . -t ansible-docgen:latest
docker run -ti --rm -v $PWD:/app ansible-docgen:latest ansible-docgen -p ansible-playbooks/roles/myrole
```

**Comment by Roman-devs** [comment]
Hi, i recently stumbled upon this issue, since i tried using ansible-docgen myself. It seems that the pyyaml version is required to be 5.4.1 or lower, since the loader is deprecated. I solved the issue by using version 5.4.1.

`pip install pyyaml==5.4.1`

 Afterwards i get a YAMLLoadWarning, however the desired documentation readme file is created by docgen in the desired role folder for now. 

`YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  yamldata = yaml.load(data)
`

Hope that helps!

**Comment by thedavidwhiteside** [comment]
Resolved in v0.0.4.
